connPFM/tests/test_ev.py

```diff
@@ -28,34 +28,110 @@ def test_rss_surr(AUC_file, atlas_file, surr_dir, rssr_auc_file):
     )
 
     AUC_img = masker.fit_transform(AUC_file)
-    _, u, v = ev.calculate_ets(AUC_img, AUC_img.shape[1])
-    rssr, _, _ = ev.rss_surr(AUC_img, u, v, join(surr_dir, "surrogate_AUC_"), "", masker, 0)
+    _, u, v = connectivity_utils.calculate_ets(AUC_img, AUC_img.shape[1])
+    rssr, _, _ = connectivity_utils.rss_surr(
+        AUC_img, u, v, join(surr_dir, "surrogate_AUC_"), "", masker, 0
+    )
     rssr_auc = np.loadtxt(rssr_auc_file)
     assert np.all(np.isclose(rssr, rssr_auc))
 
 
 def test_threshold_ets_matrix():
```

Comment:
Could you explain the logic behind this test?

Reply:
You wrote that test haha. I just moved the functions to `connectivity_utils`. But now that I see it, I think it makes more sense to join this test with the one above.